core/src/no/group15/playmagic/ecs/Factories.kt

```diff
@@ -21,6 +21,7 @@ fun engineFactory(injectContext: Context, viewport: Viewport): Engine {
 	engine.addSystem(MovementSystem(1, injectContext, gameMap))
 	engine.addSystem(CollisionSystem(2))
 	engine.addSystem(RenderingSystem(10, viewport, batch))
+	engine.addSystem(AnimationSystem(10))
```

Comment:
priority should be before rendering I guess, so 9

Reply:
ah, i misinterpreted the description on https://github.com/libgdx/ashley/wiki/How-to-use-Ashley, thought it was for performance and higher number was less important.